library.properties

```diff
@@ -0,0 +1,10 @@
+name=ESPInfluxdb
+version=1.0.0
+author=hwwong
+maintainer=hwwong
+sentence=ESP32 / ESP8266 influxdb client.
+paragraph=ESP32 / ESP8266 influxdb client, posts and queries data
```

Comment:
The Arduino IDE's Library Manager displays the text defined in the library.properties paragraph field immediately after the sentence text so it's not desirable to duplicate the sentence text in paragraph.